config/locales/ne.yml

```diff
@@ -0,0 +1,1338 @@
+    test: परीक्षण
+    test_mailer:
+      test_email:
+        greeting: बधाई छ!
+        message: यदि तपाईंले यो ईमेल प्राप्त गर्नुभयो भने, तपाईंको ईमेल सेटिंग्स सही छ।
+        subject: परीक्षण मेल
+    test_mode: परीक्षण मोड
+    thank_you_for_your_order: तपाईको व्यवसायका लागि धन्यबाद। तपाईंको रेकर्डहरूको लागि कृपया यस कन्फर्मेसन पृष्ठको प्रतिलिपि प्रिन्ट गर्नुहोस्।
+    there_are_no_items_for_this_order: यस अर्डरको लागि त्यहाँ कुनै आईटमहरू छैनन्। आदेश जारी गर्न एक वस्तु जोड्नुहोस्।
+    there_were_problems_with_the_following_fields: तलका क्षेत्रहरूसँग समस्याहरू थिए
+    this_order_has_already_received_a_refund:
+    thumbnail: थम्बनेल
+    tiered_flat_rate:
+    tiered_percent:
+    tiers:
+    time: समय
+    to_add_variants_you_must_first_define: भेरियन्टहरू थप्न, तपाईले पहिले परिभाषित गर्नु पर्छ
+    total: कुल
+    total_per_item:
+    total_pre_tax_refund:
+    total_price:
+    total_sales:
+    track_inventory:
+    tracking: ट्र्याकिंग
+    tracking_number: ट्र्याकिंग नम्बर
+    tracking_url: ट्र्याकिंग URL
+    tracking_url_placeholder: उदाहरण को लागी http://quickship.com/package?num=:tracking
+    transaction_id:
+    transfer_from_location: बाट स्थानान्तरण
+    transfer_stock: स्टक स्थानान्तरण
+    transfer_to_location: स्थानान्तरण गर्न
+    tree: रूख
+    type: प्रकार
+    type_to_search: खोजी गर्न टाइप गर्नुहोस्
+    unable_to_connect_to_gateway: गेटवेमा जडान गर्न असमर्थ।
+    unable_to_create_reimbursements:
+    under_price: '%{price} अन्तर्गत'
+    unlock: अनलक गर्नुहोस्
+    unrecognized_card_type: अपरिचित कार्ड प्रकार
+    unshippable_items: अप्रकाशित वस्तुहरू
+    update: अद्यावधिक गर्नुहोस्
+    updating: अद्यावधिक गर्दै
+    usage_limit: उपयोग सीमा
+    use_app_default:
+    use_billing_address: बिलि ठेगाना प्रयोग गर्नुहोस्
+    use_new_cc: नयाँ कार्ड प्रयोग गर्नुहोस्
+    use_s3: छविहरूका लागि Amazon S3 प्रयोग गर्नुहोस्
+    user: प्रयोगकर्ता
+    user_rule:
+      choose_users: प्रयोगकर्ता छान्नुहोस्
+    users: प्रयोगकर्ताहरू
+    validation:
+      cannot_be_less_than_shipped_units: पठाइएको इकाईहरूको संख्या भन्दा कम हुन सक्दैन।
+      cannot_destroy_line_item_as_inventory_units_have_shipped:
+      exceeds_available_stock: उपलब्ध स्टक नाघ्यो। कृपया सुनिश्चित गर्नुहोस् कि रेखा आईटमहरूमा मान्य मात्रा छ।
+      is_too_large: धेरै ठूलो छ - हातमा स्टकले अनुरोध गरिएको मात्रा कभर गर्न सक्दैन!
+      must_be_int: पूर्णांक हुनुपर्दछ
+      must_be_non_negative: एक गैर-नकारात्मक मान हुनु पर्छ
+      unpaid_amount_not_zero:
+    value: मूल्य
+    variant: भेरियन्ट
+    variant_placeholder: एक संस्करण छान्नुहोस्
+    variants: भेरियन्ट्स
+    version: संस्करण
+    void: शून्य
+    weight: तौल
+    what_is_a_cvv: (CVV) क्रेडिट कार्ड कोड के हो?
+    what_is_this: यो के हो?
+    width: चौड़ाई
+    year: बर्ष
+    you_have_no_orders_yet: तपाईंसँग अहिलेसम्म कुनै अर्डर छैन।
+    your_cart_is_empty: तपाईंको टोकरी खाली छ
+    your_order_is_empty_add_product: तपाईंको अर्डर खाली छ, कृपया खोज्नुहोस् र माथिको उत्पादन थप्नुहोस्
```

Comment:
kripaya kehi samagri थप्नुहोस्, not `कृपया खोज्नुहोस् र माथिको उत्पादन थप्नुहोस्`